app/src/store/rootSaga.js

```diff
@@ -28,7 +28,7 @@ import { membersSagas } from 'controllers/members';
 import { testItemsSagas } from 'controllers/testItem';
 import { historySagas } from 'controllers/itemsHistory';
 import { logSagas } from 'controllers/log';
-import { administrateSagas } from 'controllers/administrate';
+import { administrateSagas } from 'controllers/instance';
```

Comment:
Should be renamed to be consistent with Instance level naming.

Reply:
Done.